src/brain/issueLabelHandler/followups.ts

```diff
@@ -158,5 +166,23 @@ export async function ensureOneWaitingForLabel({
 
   await modifyProjectIssueField(itemId, labelName, STATUS_FIELD_ID, octokit);
 
+  let timeToTriageBy
+  if (labelName === WAITING_FOR_PRODUCT_OWNER_LABEL) {
+    timeToTriageBy = await calculateSLOViolationTriage(UNTRIAGED_LABEL, issue.labels) || moment().toISOString();
+  }
+  else if (labelName === WAITING_FOR_SUPPORT_LABEL) {
+    timeToTriageBy = await calculateSLOViolationRoute(UNROUTED_LABEL) || moment().toISOString()
+  }
+  else{
+    timeToTriageBy = "";
```

Comment:
This is where we'll bring in stalebot, eh?

Reply:
Not necessarily needed actually, GH has something neat that includes the `updated_at` property for issues that tells us when any action was last done on an issue. This is just there to indicate there isn't a respond by date, since we're just waiting for community to respond.

See this for the usage of the `updated_at` property
https://github.com/getsentry/eng-pipes/blob/main/src/webhooks/pubsub/stalebot.ts#L82